kernel/kernel-api/src/main/java/io/delta/kernel/internal/ScanBuilderImpl.java

```diff
@@ -41,27 +39,29 @@
 public class ScanBuilderImpl
     implements ScanBuilder {
 
+    private final Path dataPath;
```

Comment:
As the PR description states, I have changed the order of the definition of member variables to match the constructor order